### ISSUE TITLE
2025-06-02, Papers for large language model, llm

### ISSUE THREAD

**Issue by 9tailwolf** [Arxiv-Sender/Arxiv-Sender#188]
# Today's Arxiv Papers

(1) [Open CaptchaWorld: A Comprehensive Web-based Platform for Testing and Benchmarking Multimodal LLM Agents](http://arxiv.org/abs/2505.24878v1), 2025-05-30

(2) [MoDoMoDo: Multi-Domain Data Mixtures for Multimodal LLM Reinforcement Learning](http://arxiv.org/abs/2505.24871v1), 2025-05-30

(3) [SiLVR: A Simple Language-based Video Reasoning Framework](http://arxiv.org/abs/2505.24869v1), 2025-05-30

(4) [ProRL: Prolonged Reinforcement Learning Expands Reasoning Boundaries in Large Language Models](http://arxiv.org/abs/2505.24864v1), 2025-05-30

(5) [MetaFaith: Faithful Natural Language Uncertainty Expression in LLMs](http://arxiv.org/abs/2505.24858v1), 2025-05-30

(6) [Harnessing Negative Signals: Reinforcement Distillation from Teacher Data for LLM Reasoning](http://arxiv.org/abs/2505.24850v1), 2025-05-30

(7) [MiCRo: Mixture Modeling and Context-aware Routing for Personalized Preference Learning](http://arxiv.org/abs/2505.24846v1), 2025-05-30

(8) [Chameleon: A Flexible Data-mixing Framework for Language Model Pretraining and Finetuning](http://arxiv.org/abs/2505.24844v1), 2025-05-30

(9) [Cascading Adversarial Bias from Injection to Distillation in Language Models](http://arxiv.org/abs/2505.24842v1), 2025-05-30

(10) [Vision LLMs Are Bad at Hierarchical Visual Understanding, and LLMs Are the Bottleneck](http://arxiv.org/abs/2505.24840v1), 2025-05-30

(11) [VideoCAD: A Large-Scale Video Dataset for Learning UI Interactions and 3D Reasoning from CAD Software](http://arxiv.org/abs/2505.24838v1), 2025-05-30

(12) [Improving Reliability and Explainability of Medical Question Answering through Atomic Fact Checking in Retrieval-Augmented LLMs](http://arxiv.org/abs/2505.24830v1), 2025-05-30

(13) [LegalEval-Q: A New Benchmark for The Quality Evaluation of LLM-Generated Legal Text](http://arxiv.org/abs/2505.24826v1), 2025-05-30

(14) [PhySense: Principle-Based Physics Reasoning Benchmarking for Large Language Models](http://arxiv.org/abs/2505.24823v1), 2025-05-30

(15) [Guiding Generative Storytelling with Knowledge Graphs](http://arxiv.org/abs/2505.24803v1), 2025-05-30

(16) [Drop Dropout on Single-Epoch Language Model Pretraining](http://arxiv.org/abs/2505.24788v1), 2025-05-30

(17) [Draw ALL Your Imagine: A Holistic Benchmark and Agent Framework for Complex Instruction-based Image Generation](http://arxiv.org/abs/2505.24787v1), 2025-05-30

(18) [EXP-Bench: Can AI Conduct AI Research Experiments?](http://arxiv.org/abs/2505.24785v1), 2025-05-30

(19) [Revisiting Epistemic Markers in Confidence Estimation: Can Markers Accurately Reflect Large Language Models' Uncertainty?](http://arxiv.org/abs/2505.24778v1), 2025-05-30

(20) [AFLoRA: Adaptive Federated Fine-Tuning of Large Language Models with Resource-Aware Low-Rank Adaption](http://arxiv.org/abs/2505.24773v1), 2025-05-30

(21) [From Macro to Micro: Probing Dataset Diversity in Language Model Fine-Tuning](http://arxiv.org/abs/2505.24768v1), 2025-05-30

(22) [A survey of using EHR as real-world evidence for discovering and validating new drug indications](http://arxiv.org/abs/2505.24767v1), 2025-05-30

(23) [LGAR: Zero-Shot LLM-Guided Neural Ranking for Abstract Screening in Systematic Literature Reviews](http://arxiv.org/abs/2505.24757v1), 2025-05-30

(24) [SUMO: Subspace-Aware Moment-Orthogonalization for Accelerating Memory-Efficient LLM Training](http://arxiv.org/abs/2505.24749v1), 2025-05-30

(25) [Circuit Stability Characterizes Language Model Generalization](http://arxiv.org/abs/2505.24731v1), 2025-05-30

(26) [Reflect, Retry, Reward: Self-Improving LLMs via Reinforcement Learning](http://arxiv.org/abs/2505.24726v1), 2025-05-30

(27) [Talking Transactions: Decentralized Communication through Ethereum Input Data Messages (IDMs)](http://arxiv.org/abs/2505.24724v1), 2025-05-30

(28) [HELM: Hyperbolic Large Language Models via Mixture-of-Curvature Experts](http://arxiv.org/abs/2505.24722v1), 2025-05-30

(29) [Reinforcing Video Reasoning with Focused Thinking](http://arxiv.org/abs/2505.24718v1), 2025-05-30

(30) [Towards Scalable Schema Mapping using Large Language Models](http://arxiv.org/abs/2505.24716v1), 2025-05-30

(31) [FinMME: Benchmark Dataset for Financial Multi-Modal Reasoning Evaluation](http://arxiv.org/abs/2505.24714v1), 2025-05-30

(32) [HESEIA: A community-based dataset for evaluating social biases in large language models, co-designed in real school settings in Latin America](http://arxiv.org/abs/2505.24712v1), 2025-05-30

(33) [Causal-aware Large Language Models: Enhancing Decision-Making Through Learning, Adapting and Acting](http://arxiv.org/abs/2505.24710v1), 2025-05-30

(34) [Multi-Domain ABSA Conversation Dataset Generation via LLMs for Real-World Evaluation and Model Comparison](http://arxiv.org/abs/2505.24701v1), 2025-05-30

(35) [Speech-to-Text Translation with Phoneme-Augmented CoT: Enhancing Cross-Lingual Transfer in Low-Resource Scenarios](http://arxiv.org/abs/2505.24691v1), 2025-05-30

(36) [BPE Stays on SCRIPT: Structured Encoding for Robust Multilingual Pretokenization](http://arxiv.org/abs/2505.24689v1), 2025-05-30

(37) [Soft Reasoning: Navigating Solution Spaces in Large Language Models through Controlled Embedding Exploration](http://arxiv.org/abs/2505.24688v1), 2025-05-30

(38) [Should I Share this Translation? Evaluating Quality Feedback for User Reliance on Machine Translation](http://arxiv.org/abs/2505.24683v1), 2025-05-30

(39) [A Simple Linear Patch Revives Layer-Pruned Large Language Models](http://arxiv.org/abs/2505.24680v1), 2025-05-30

(40) [TRIDENT: Enhancing Large Language Model Safety with Tri-Dimensional Diversified Red-Teaming Data Synthesis](http://arxiv.org/abs/2505.24672v1), 2025-05-30

(41) [Multiple LLM Agents Debate for Equitable Cultural Alignment](http://arxiv.org/abs/2505.24671v1), 2025-05-30

(42) [Can LLMs and humans be friends? Uncovering factors affecting human-AI intimacy formation](http://arxiv.org/abs/2505.24658v1), 2025-05-30

(43) [Adaptable Cardiovascular Disease Risk Prediction from Heterogeneous Data using Large Language Models](http://arxiv.org/abs/2505.24655v1), 2025-05-30

(44) [Are Optimal Algorithms Still Optimal? Rethinking Sorting in LLM-Based Pairwise Ranking with Batching and Caching](http://arxiv.org/abs/2505.24643v1), 2025-05-30

(45) [Efficient Text Encoders for Labor Market Analysis](http://arxiv.org/abs/2505.24640v1), 2025-05-30

(46) [Disentangling Language and Culture for Evaluating Multilingual Large Language Models](http://arxiv.org/abs/2505.24635v1), 2025-05-30

(47) [The Hallucination Dilemma: Factuality-Aware Reinforcement Learning for Large Reasoning Models](http://arxiv.org/abs/2505.24630v1), 2025-05-30

(48) [Learning from Videos for 3D World: Enhancing MLLMs with 3D Vision Geometry Priors](http://arxiv.org/abs/2505.24625v1), 2025-05-30

(49) [Random Rule Forest (RRF): Interpretable Ensembles of LLM-Generated Questions for Predicting Startup Success](http://arxiv.org/abs/2505.24622v1), 2025-05-30

(50) [Benchmarking Large Language Models for Cryptanalysis and Mismatched-Generalization](http://arxiv.org/abs/2505.24621v1), 2025-05-30

(51) [Eye of Judgement: Dissecting the Evaluation of Russian-speaking LLMs with POLLUX](http://arxiv.org/abs/2505.24616v1), 2025-05-30

(52) [Harnessing Large Language Models for Scientific Novelty Detection](http://arxiv.org/abs/2505.24615v1), 2025-05-30

(53) [When Harry Meets Superman: The Role of The Interlocutor in Persona-Based Dialogue Generation](http://arxiv.org/abs/2505.24613v1), 2025-05-30

(54) [Mixture-of-Experts for Personalized and Semantic-Aware Next Location Prediction](http://arxiv.org/abs/2505.24597v1), 2025-05-30

(55) [NexusSum: Hierarchical LLM Agents for Long-Form Narrative Summarization](http://arxiv.org/abs/2505.24575v1), 2025-05-30

(56) [Bench4KE: Benchmarking Automated Competency Question Generation](http://arxiv.org/abs/2505.24554v1), 2025-05-30

(57) [CREFT: Sequential Multi-Agent LLM for Character Relation Extraction](http://arxiv.org/abs/2505.24553v1), 2025-05-30

(58) [Cross-Attention Speculative Decoding](http://arxiv.org/abs/2505.24544v1), 2025-05-30

(59) [Mixpert: Mitigating Multimodal Learning Conflicts with Efficient Mixture-of-Vision-Experts](http://arxiv.org/abs/2505.24541v1), 2025-05-30

(60) [Localizing Persona Representations in LLMs](http://arxiv.org/abs/2505.24539v1), 2025-05-30

(61) [Don't Erase, Inform! Detecting and Contextualizing Harmful Language in Cultural Heritage Collections](http://arxiv.org/abs/2505.24538v1), 2025-05-30

(62) [Beyond Linear Steering: Unified Multi-Attribute Control for Language Models](http://arxiv.org/abs/2505.24535v1), 2025-05-30

(63) [DEEPQUESTION: Systematic Generation of Real-World Challenges for Evaluating LLMs Performance](http://arxiv.org/abs/2505.24532v1), 2025-05-30

(64) [Stress-testing Machine Generated Text Detection: Shifting Language Models Writing Style to Fool Detectors](http://arxiv.org/abs/2505.24523v1), 2025-05-30

(65) [un$^2$CLIP: Improving CLIP's Visual Detail Capturing Ability via Inverting unCLIP](http://arxiv.org/abs/2505.24517v1), 2025-05-30

(66) [Can Slow-thinking LLMs Reason Over Time? Empirical Studies in Time Series Forecasting](http://arxiv.org/abs/2505.24511v1), 2025-05-30

(67) [TimeHC-RL: Temporal-aware Hierarchical Cognitive Reinforcement Learning for Enhancing LLMs' Social Intelligence](http://arxiv.org/abs/2505.24500v1), 2025-05-30

(68) [Reason-SVG: Hybrid Reward RL for Aha-Moments in Vector Graphics Generation](http://arxiv.org/abs/2505.24499v1), 2025-05-30

(69) [MELT: Towards Automated Multimodal Emotion Data Annotation by Leveraging LLM Embedded Knowledge](http://arxiv.org/abs/2505.24493v1), 2025-05-30

(70) [Towards Effective Code-Integrated Reasoning](http://arxiv.org/abs/2505.24480v1), 2025-05-30

(71) [Leveraging Knowledge Graphs and LLMs for Structured Generation of Misinformation](http://arxiv.org/abs/2505.24479v1), 2025-05-30

(72) [Optimizing the Interface Between Knowledge Graphs and LLMs for Complex Reasoning](http://arxiv.org/abs/2505.24478v1), 2025-05-30

(73) [Period-LLM: Extending the Periodic Capability of Multimodal Large Language Model](http://arxiv.org/abs/2505.24476v1), 2025-05-30

(74) [VietMix: A Naturally Occurring Vietnamese-English Code-Mixed Corpus with Iterative Augmentation for Machine Translation](http://arxiv.org/abs/2505.24472v1), 2025-05-30

(75) [SA-Person: Text-Based Person Retrieval with Scene-aware Re-ranking](http://arxiv.org/abs/2505.24466v1), 2025-05-30

(76) [SEAR: A Multimodal Dataset for Analyzing AR-LLM-Driven Social Engineering Behaviors](http://arxiv.org/abs/2505.24458v1), 2025-05-30

(77) [LPASS: Linear Probes as Stepping Stones for vulnerability detection using compressed LLMs](http://arxiv.org/abs/2505.24451v1), 2025-05-30

(78) [When Large Multimodal Models Confront Evolving Knowledge:Challenges and Pathways](http://arxiv.org/abs/2505.24449v1), 2025-05-30

(79) [Exploring the Impact of Occupational Personas on Domain-Specific QA](http://arxiv.org/abs/2505.24448v1), 2025-05-30

(80) [Learning Safety Constraints for Large Language Models](http://arxiv.org/abs/2505.24445v1), 2025-05-30

(81) [RMoA: Optimizing Mixture-of-Agents through Diversity Maximization and Residual Compensation](http://arxiv.org/abs/2505.24442v1), 2025-05-30

(82) [SORCE: Small Object Retrieval in Complex Environments](http://arxiv.org/abs/2505.24441v1), 2025-05-30

(83) [Model Unlearning via Sparse Autoencoder Subspace Guided Projections](http://arxiv.org/abs/2505.24428v1), 2025-05-30

(84) [MMAFFBen: A Multilingual and Multimodal Affective Analysis Benchmark for Evaluating LLMs and VLMs](http://arxiv.org/abs/2505.24423v1), 2025-05-30

(85) [LLMs Are Globally Multilingual Yet Locally Monolingual: Exploring Knowledge Transfer via Language and Thought Theory](http://arxiv.org/abs/2505.24409v1), 2025-05-30

(86) [ClueAnchor: Clue-Anchored Knowledge Reasoning Exploration and Optimization for Retrieval-Augmented Generation](http://arxiv.org/abs/2505.24388v1), 2025-05-30

(87) [Breaking the Gold Standard: Extracting Forgotten Data under Exact Unlearning in Large Language Models](http://arxiv.org/abs/2505.24379v1), 2025-05-30

(88) [LLM Inference Enhanced by External Knowledge: A Survey](http://arxiv.org/abs/2505.24377v1), 2025-05-30

(89) [Grid-LOGAT: Grid Based Local and Global Area Transcription for Video Question Answering](http://arxiv.org/abs/2505.24371v1), 2025-05-30

(90) [Adversarial Preference Learning for Robust LLM Alignment](http://arxiv.org/abs/2505.24369v1), 2025-05-30

(91) [Knowing Before Saying: LLM Representations Encode Information About Chain-of-Thought Success Before Completion](http://arxiv.org/abs/2505.24362v1), 2025-05-30

(92) [ReCalKV: Low-Rank KV Cache Compression via Head Reordering and Offline Calibration](http://arxiv.org/abs/2505.24357v1), 2025-05-30

(93) [Unifying Language Agent Algorithms with Graph-based Orchestration Engine for Reproducible Agent Research](http://arxiv.org/abs/2505.24354v1), 2025-05-30

(94) [Fewer Hallucinations, More Verification: A Three-Stage LLM-Based Framework for ASR Error Correction](http://arxiv.org/abs/2505.24347v1), 2025-05-30

(95) [Exploring Multimodal Challenges in Toxic Chinese Detection: Taxonomy, Benchmark, and Findings](http://arxiv.org/abs/2505.24341v1), 2025-05-30

(96) [GeoVision Labeler: Zero-Shot Geospatial Classification with Vision and Language Models](http://arxiv.org/abs/2505.24340v1), 2025-05-30

(97) [Pangu DeepDiver: Adaptive Search Intensity Scaling via Open-Web Reinforcement Learning](http://arxiv.org/abs/2505.24332v1), 2025-05-30

(98) [Context-Aware Sentiment Forecasting via LLM-based Multi-Perspective Role-Playing Agents](http://arxiv.org/abs/2505.24331v1), 2025-05-30

(99) [DisTime: Distribution-based Time Representation for Video Large Language Models](http://arxiv.org/abs/2505.24329v1), 2025-05-30

(100) [SwiftEval: Developing a Language-Specific Benchmark for LLM-generated Code Evaluation](http://arxiv.org/abs/2505.24324v1), 2025-05-30
